Library/Formula/bip.rb

```diff
@@ -1,15 +1,31 @@
-require "formula"
-
 class Bip < Formula
-  homepage "http://bip.milkypond.org"
-  url "http://ftp.de.debian.org/debian/pool/main/b/bip/bip_0.8.9.orig.tar.gz"
+  # Auto redirects to https but currently has broken/untrusted warning.
+  # homepage "https://bip.milkypond.org"
```

Comment:
Sending them to the debian page when they want to go to the actual homepage isn't a great experience either :/